cohort-evaluator-spark/src/main/java/com/ibm/cohort/cql/spark/metadata/BaseOutputMetadataWriter.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.cohort.cql.spark.metadata;
+
+
+import org.apache.commons.collections.CollectionUtils;
+
+public abstract class BaseOutputMetadataWriter {
```

Comment:
I was expecting to find an OutputMetadataWriter interface implemented here. Did you decide against that?

Reply:
I still couldn't figure out what made sense to  have in the interface.  Do you have a concrete idea?

Reply:
I added an interface that has `void writeMetadata(EvaluationSummary evaluationSummary);`. Wasn't sure if you had more than that in mind.

Reply:
I am satisfied with that.